CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/55654104/114203099-2319d780-9958-11eb-8d57-5533c70b3859.png)
There is no padding at the bottom on mobile view. I think this would look better.